### ISSUE TITLE
Is this library fit for android 

### ISSUE THREAD

**Issue by ohmerhe** [jhalterman/typetools#18]
I try to use this library in android, but get exception in this line 

```
SUPPORTS_LAMBDAS = Double.valueOf(System.getProperty("java.version").substring(0, 3)) >= 1.8;
```
`System.getProperty("java.version")` return "0" which can not be used to next action.

**Comment by jhalterman** [comment]
It has not been tested against Android. I suspect that much of TypeTools would (or could be made to) work on Android, though the lambda/method reference stuff probably would not. Thanks for filing this.

**Comment by ohmerhe** [comment]
Yes, it would not be strenuous to support android platform.

**Comment by jhalterman** [comment]
Fixed via #20

**Comment by lvtanxi** [comment]
ConstantPool this method is in sun's JDK, while Android is not.
